docs/installation_guide.rst

```diff
@@ -169,58 +169,143 @@ You can now start Rotki::
+4. Recalling the instructions above on how to set up the virtualenv, set one up for rotki (you don't have to use the name ``rotki-develop``)::
 
+    mkvirtualenv rotki-develop
 
-Then you can go to ``pysqlcipher3`` and edit the ``setup.py`` file to include the location of the headers and the libraries you just compiled. Add the following to the extension that is being built.::
+5. Ensure that you're in the directory where you downloaded/cloned the rotki source and bind the virtualenv to the directory::
 
+    setprojectdir .
 
-    library_dirs=[r'C:\Users\yourusername\w\sqlcipher'],
-    include_dirs=[r'C:\Users\yourusername\w'],
+If at any time you want to dissasociate with the virtual env, you can use the command ``deactivate``. Whenever you open a new terminal you can now use ``workon rotki-develop`` (if you named your virtualenv something else then use that instead of ``rotki-develop``) and it should establish the link to the python virtualenv you created and set your working directory to the directory you were in in Step 5. Following the example above, if you open a brand new terminal and type in ``workon rotki-deveop`` your terminal prompt should look something like::
+
+    (rotki-develop) c:\dev\rotki-develop>
+
+6. Now it's time to install all the python requirements. In the open terminal with your virtualenv activated, execute::
+
+    install -r requirements_dev.txt
+
+Pay close attention to the results of the command. Sometimes modules are reported as successfully installed but in reality the build has failed. You should carefully scroll through the buffer to ensure everything has been built & installed correct. 
+
+At this point, it's likely that pysqlcipher3 has not been built and installed correctly, and you will need to install it manually. If pysqlcipher3 installed successfully, you can skip Steps 7 - 9 and move on to the next section.
+
+7. Go back to your development directory and download / clone (if you want to use git but don't want to clone the whole project, check out `degit <https://www.npmjs.com/package/degit>`_) the source for `pysqlcipher3 <https://github.com/rigglemania/pysqlcipher3>`_.
+
+8. With your code editor of choice, edit the ``setup.py`` file in ``\pysqlcipher3\``, make the following changes and save the file::
+
+    Lines 165 - 169 Before
+            ext_modules=[Extension(
+            name=PACKAGE_NAME + EXTENSION_MODULE_NAME,
+            sources=sources,
+            define_macros=define_macros)
+        ],
+    Lines 165 ... AFTER
+            ext_modules=[Extension(
+            name=PACKAGE_NAME + EXTENSION_MODULE_NAME,
+            sources=sources,
+            library_dirs=[r'<DIRECTORY WHERE YOU BUILT SQLCIPHER TO (i.e. where the compiled sqlcipher.exe and sqlcipher.dll are)>'],
+	        include_dirs=[r'<THE PARENT DIRECTORY OF THE ABOVE DIRECTORY>'],
```

Comment:
can you make sure the indentation of the `include_dirs` and the `library_dirs` are the same?

Reply:
Fixed.